runtime/include/j9memcategories.h

```diff
@@ -79,5 +79,8 @@
 #define J9MEM_CATEGORY_UNUSED37 37
 #define J9MEM_CATEGORY_SUN_MISC_UNSAFE_ALLOCATEDBB 38
 #define J9MEM_CATEGORY_MODULES 39
+#if JAVA_SPEC_VERSION >= 16
+#define J9MEM_CATEGORY_VM_FFI 40
```

Comment:
This could use one of the unused numbers.

Reply:
Replaced `J9MEM_CATEGORY_UNUSED1` with `J9MEM_CATEGORY_VM_FFI` as suggested.